boards/pic32-clicker/doc.txt

```diff
@@ -3,16 +3,91 @@
 @ingroup     boards
 @brief       Support for the MikroE PIC32 Clicker
 
-For instructions on reflashing see:
+## Overview
 
-https://docs.creatordev.io/clicker/guides/quick-start-guide/#programming-a-6lowpan-clicker
-
-The RIOT build generates a hexfile compatible with MPLAB-IPE.
+The `6lowpan clicker` is an evaluation board by Mikroe featuring a
+PIC32MX470F512H, a 6lowpan radio (CA810) by Cascoda and a Mikrobus
+socket for expansion boards.
 
 More general information on the board and related documentation can be found
-here:
+here: https://www.mikroe.com/clicker-6lowpan. Schematics for the board can be
```

Comment:
for readability and *nicer* HTML output: please use proper Markdown links, i.e. [here](<url>) or [manual](<url>) below.

Reply:
Fixed